clients/go/apis/v1alpha1/config.go

```diff
@@ -114,11 +114,15 @@ func (cfg *Config) Validate() error {
 
 // SetDefault sets default for the config.
 func (cfg *Config) SetDefault() {
-	// TODO: set defaults for other fields if necessary.
 	if cfg.Version == "" {
 		cfg.Version = Version
 	}
 
+	// Default log mode to "fail close".
+	if cfg.LogMode == "" || cfg.LogMode == "LOG_MODE_UNSPECIFIED" {
+		cfg.LogMode = "FAIL_CLOSE"
```

Comment:
These are feeling like they should be enums or constants in the protos.

Reply:
Yes they are. Fixed.